feature/experimental/policy/policy_vrf_selection/otg_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -313,33 +317,42 @@ func configureATE(t *testing.T, ate *ondatra.ATEDevice) *trafficFlows {
 	// Create traffic flows
 	t.Logf("*** Configuring OTG flows ...")
 	topo.Flows().Clear().Items()
-	ipInIPFlow1 := createFlow("ipInIPFlow1", topo, ateDst, "IPv4")
-	ipInIPFlow2 := createFlow("ipInIPFlow2", topo, ateDst2, "IPv4")
-	ipInIPFlow3 := createFlow("ipInIPFlow3", topo, ateDst, "IPv4")
-	ipInIPFlow4 := createFlow("ipInIPFlow4", topo, ateDst2, "IPv4")
-	ipv6InIPFlow5 := createFlow("ipv6InIPFlow5", topo, ateDst, "IPv6")
-	ipv6InIPFlow6 := createFlow("ipv6InIPFlow6", topo, ateDst2, "IPv6")
-	ipv6InIPFlow7 := createFlow("ipv6InIPFlow7", topo, ateDst, "IPv6")
-	ipv6InIPFlow8 := createFlow("ipv6InIPFlow8", topo, ateDst2, "IPv6")
-	nativeIPv4 := createFlow("nativeIPv4", topo, ateDst2, "")
-	nativeIPv6 := createFlow("nativeIPv6", topo, ateDst2, "")
+	ipInIPFlow1 := createIPv4Flow("ipInIPFlow1", topo, ateDst, nonMatchingIpv4Address, ateDestIPv4VLAN10, "IPv4")
+	ipInIPFlow2 := createIPv4Flow("ipInIPFlow2", topo, ateDst2, nonMatchingIpv4Address, ateDestIPv4VLAN20, "IPv4")
+	ipInIPFlow3 := createIPv4Flow("ipInIPFlow3", topo, ateDst, srcIpv4Address, ateDestIPv4VLAN10, "IPv4")
+	ipInIPFlow4 := createIPv4Flow("ipInIPFlow4", topo, ateDst2, srcIpv4Address, ateDestIPv4VLAN20, "IPv4")
+	ipv6InIPFlow5 := createIPv4Flow("ipv6InIPFlow5", topo, ateDst, nonMatchingIpv4Address, ateDestIPv4VLAN10, "IPv6")
+	ipv6InIPFlow6 := createIPv4Flow("ipv6InIPFlow6", topo, ateDst2, nonMatchingIpv4Address, ateDestIPv4VLAN20, "IPv6")
+	ipv6InIPFlow7 := createIPv4Flow("ipv6InIPFlow7", topo, ateDst, srcIpv4Address, ateDestIPv4VLAN10, "IPv6")
+	ipv6InIPFlow8 := createIPv4Flow("ipv6InIPFlow8", topo, ateDst2, srcIpv4Address, ateDestIPv4VLAN20, "IPv6")
+	nativeIPv4 := createIPv4Flow("nativeIPv4", topo, ateDst2, ateSrc.IPv4, ateDestIPv4VLAN20, "")
+	nativeIPv6 := topo.Flows().Add().SetName("nativeIPv6")
+	nativeIPv6.Metrics().SetEnable(true)
+	nativeIPv6.TxRx().Device().SetTxNames([]string{ateSrc.Name + ".IPv6"}).SetRxNames([]string{ateDst.Name + ".IPv6"})
+	nativeIPv6.Packet().Add().Ethernet().Src().SetValue(ateSrc.MAC)
+	v6 := nativeIPv6.Packet().Add().Ipv6()
+	v6.Src().SetValue(ateSrc.IPv6)
+	v6.Dst().SetValue("2001:DB8:2::")
+	nativeIPv6.Size().SetFixed(512)
+	nativeIPv6.Rate().SetChoice("percentage").SetPercentage(5)
 
 	t.Logf("Pushing config to ATE and starting protocols...")
 	ate.OTG().PushConfig(t, topo)
 	ate.OTG().StartProtocols(t)
+	otgutils.WaitForARP(t, ate.OTG(), topo, "IPv4")
+	otgutils.WaitForARP(t, ate.OTG(), topo, "IPv6")
 	return &trafficFlows{ipInIPFlow1, ipInIPFlow2, ipInIPFlow3, ipInIPFlow4, ipv6InIPFlow5, ipv6InIPFlow6, ipv6InIPFlow7, ipv6InIPFlow8, nativeIPv4, nativeIPv6}
 }
 
-func createFlow(name string, top gosnappi.Config, dst attrs.Attributes, innerIpType string) gosnappi.Flow {
-
+func createIPv4Flow(name string, top gosnappi.Config, dst attrs.Attributes, srcIP, dstIP, innerIpType string) gosnappi.Flow {
```

Comment:
Remove `dst attrs.Attributes`? Not used anymore.

Reply:
Still used when setting the rx endpoint for the flow: https://github.com/openconfig/featureprofiles/pull/1771/files#diff-2dbe3c4b22b5dbf32c87c8c13d9a3074d3017ea2d2b2277931c48402014ba152R359